create_cluster.py

```diff
@@ -239,6 +239,34 @@ def get_subnets(prefix_filter: str, regions: list) -> dict:
     return subnets
 
 
+def setup_dns_records(cluster_name: str, hosted_zone: str, node_ips: dict):
+    r53 = boto3.client('route53')
+
+    zone = None
+    zones = r53.list_hosted_zones_by_name(DNSName=hosted_zone)
+    for z in zones['HostedZones']:
+        if z['Name'] == hosted_zone:
+            zone = z
+    if not zone:
+        raise Exception('Failed to find Hosted Zone {}'.format(hosted_zone))
+
+    for region, ips in node_ips.items():
+        with Action('Setting up Route53 SRV records in {}..'.format(region)):
+            r53.change_resource_record_sets(
+                HostedZoneId=zone['Id'],
+                ChangeBatch={
+                    'Changes': [{
+                        'Action': 'UPSERT',
+                        'ResourceRecordSet': {
+                            'Name': '_{}-{}._tcp.{}'.format(cluster_name, region, hosted_zone),
```

Comment:
I would add a regex validator for the `cluster_name` click argument (proposed regex: `[a-z][a-z0-9-][a-z0-9]`)